crates/bevy_render/src/lib.rs

```diff
@@ -274,6 +276,9 @@ impl Plugin for RenderPlugin {
                         .unwrap();
                     cleanup.run(&mut render_app.world);
 
+                    render_app.world.resource_mut::<ReserveCount>().0 =
```

Comment:
And here.

Reply:
Fixed in 409f644eff2ec32706b70574b926f856d2bf142c